src/Magento/FunctionalTestingFramework/Util/Path/FilePathFormatter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\Util\Path;
+
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+
+class FilePathFormatter implements FileUrlFormatterInterface
+{
+    /**
+     * Return formatted full file path from input string, or false on error
+     *
+     * @param string  $path
+     * @param boolean $withTrailingSeparator
+     * @return string
+     * @throws TestFrameworkException
+     */
+    public static function format($path, $withTrailingSeparator = false)
```

Comment:
I do not think that this function should have an option for trailing separator. In my eyes, the point of this is that it will *always* return a standardized file path, so *always* with/without trailing separator.

Reply:
Same applies to URL (didn't see it at first)

Reply:
Personally I prefer this as a optional parameter with default to true. Further code refactoring shows that about 90% of time we need the /. With this as an option gives the flexibility to choose and eliminates the use of trim() in calling function.

Reply:
I don't mind the argument as long as it defaults to the most common use case.

Right now it defaults to false but @jilu1 says she wants it to default to true. So will you be changing it to true @jilu1 ?

Reply:
@tomreece Yes. see updated version

Reply:
Cool, I'm fine with the argument if it's set to default usecase (which you both agree on).